src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -71,6 +71,12 @@ def create_connectedk8s(cmd, client, resource_group_name, cluster_name, correlat
                         distribution_version=None, azure_hybrid_benefit=None, yes=False, container_log_path=None):
     logger.warning("This operation might take a while...\n")
 
+    try:
+        if cmd.cli_ctx and hasattr(cmd.cli_ctx, 'config'):
```

Comment:
let's add one small comment here on what this is doing - i.e changing telemetry push config